src/main/java/com/g4s8/ghman/utility/EnvironmentVariables.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.utility;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Environment Variables Retriever.
+ *
+ * @since 1.0
+ * @todo #5: Implement unit tests for this class.
```

Comment:
@samihibrahim pdd puzzles are supposed to have 10 world min in this project, it's configured in `.pdd` file. And please next time try to implement unit test together with the new functionality.

Reply:
@olenagerasimova fixed the puzzle duration and surely will do.

Reply:
@olenagerasimova actually, now removed the puzzle and added the unit tests.